src/app-layer-ssl.c

```diff
@@ -1913,6 +1914,38 @@ static int SSLv3ParseHandshakeProtocol(SSLState *ssl_state, const uint8_t *input
     return (int)(input - initial_input);
 }
 
+/**
+ * \internal
+ * \brief TLS Alert parser
+ *
+ * \param sslstate  Pointer to the SSL state.
+ * \param input     Pointer to the received input data.
+ * \param input_len Length in bytes of the received data.
+ * \param direction 1 toclient, 0 toserver
+ *
+ * \retval The number of bytes parsed on success, 0 if nothing parsed, -1 on failure.
+ */
+static int SSLv3ParseAlertProtocol(
```

Comment:
So, this fn is essentially to validate the alert record and set event/applayer state based on the length and the log level. Do you plan to add more here? By the name, I thought we'd be parsing rest of the input data or something..